src/main/java/bhs/devilbotz/Robot.java

```diff
@@ -230,6 +237,7 @@ private static String getRobotConfigFilePath() throws Exception {
    *     original code</a>
    */
   private static String getMacAddress() {
+    /*
```

Comment:
Agreed. I'm fixing this right now.

Reply:
I think using the mac address is more reliable.  Using jumpers is likely to cause more confusion since we'd be relying on the electrical team to connect them properly.  The mac address would only change if we swapped the actual robo rio unit.

The issue we ran across at competition was the radio is disabled, so the network adapter doesn't get an IP address and so the mac address is null.  I fixed this in the recent commit by defaulting to the competition bot if the mac address cannot be read for whatever reason.